server/sonar-server/src/test/java/org/sonar/server/custommeasure/ws/DeleteActionTest.java

```diff
@@ -0,0 +1,129 @@
+/*
+ * SonarQube, open source software quality management tool.
+ * Copyright (C) 2008-2014 SonarSource
+ * mailto:contact AT sonarsource DOT com
+ *
+ * SonarQube is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * SonarQube is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+package org.sonar.server.custommeasure.ws;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.sonar.api.web.UserRole;
+import org.sonar.core.component.ComponentDto;
+import org.sonar.core.custommeasure.db.CustomMeasureDto;
+import org.sonar.core.permission.GlobalPermissions;
+import org.sonar.core.persistence.DbSession;
+import org.sonar.core.persistence.DbTester;
+import org.sonar.server.component.ComponentTesting;
+import org.sonar.server.component.db.ComponentDao;
+import org.sonar.server.custommeasure.persistence.CustomMeasureDao;
+import org.sonar.server.db.DbClient;
+import org.sonar.server.exceptions.ForbiddenException;
+import org.sonar.server.exceptions.ServerException;
+import org.sonar.server.tester.UserSessionRule;
+import org.sonar.server.ws.WsTester;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.sonar.server.custommeasure.persistence.CustomMeasureTesting.newCustomMeasureDto;
+import static org.sonar.server.custommeasure.ws.DeleteAction.PARAM_ID;
+
+public class DeleteActionTest {
```

Comment:
Hello Teryk,

I would like to contribute some codes  to Sonarqube project, more exactly to sonar-ws-client . The version on which I work is 5.0-SNAPSHOT. If I am allowed, could you tell me how can I commit my codes ?


Best wishes

LIN Guangrun







LIN Guangrun
guangrun.lin@gmail.com

From: Teryk Bellahsene
Date: 2015-06-08 11:41
To: SonarSource/sonarqube
Subject: Re: [sonarqube] SONAR-6616 WS custom_measures/delete (#358)
In server/sonar-server/src/test/java/org/sonar/server/custommeasure/ws/DeleteActionTest.java:
> +import org.sonar.core.persistence.DbSession;
> +import org.sonar.core.persistence.DbTester;
> +import org.sonar.server.component.ComponentTesting;
> +import org.sonar.server.component.db.ComponentDao;
> +import org.sonar.server.custommeasure.persistence.CustomMeasureDao;
> +import org.sonar.server.db.DbClient;
> +import org.sonar.server.exceptions.ForbiddenException;
> +import org.sonar.server.exceptions.ServerException;
> +import org.sonar.server.tester.UserSessionRule;
> +import org.sonar.server.ws.WsTester;
> +
> +import static org.assertj.core.api.Assertions.assertThat;
> +import static org.sonar.server.custommeasure.persistence.CustomMeasureTesting.newCustomMeasureDto;
> +import static org.sonar.server.custommeasure.ws.DeleteAction.PARAM_ID;
> +
> +public class DeleteActionTest {

you mean org.sonar.server.custommeasure.ws.DeleteActionTest#fail_when_not_found_in_db ?
—
Reply to this email directly or view it on GitHub.